moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -299,6 +299,19 @@ export const ProjectsListViewFiltersConf = {
         "string_does_not_equal_case_sensitive",
       ],
     },
+    {
+      name: "added_by",
+      label: "Created by",
+      placeholder: "Created by",
+      type: "string",
+      operators: [
+        "string_contains_case_insensitive",
+        "string_begins_with_case_insensitive",
+        "string_ends_with_case_insensitive",
+        "string_equals_case_sensitive",
+        "string_does_not_equal_case_sensitive",
+      ],
+    },
```

Comment:
this is out of scope, but it would be nice if we could use the `is` operator autocomplete to pick a user. let me know @chiaberry if you'd prefer to see that in a separate issue.

Reply:
Agreed, that would be nice to have for sure!

Reply:
We should spin this into its own issue, its a little more involved to get the autocomplete to have the list of employee first name and last names

I noticed we also have the "is" option for Team, but there is no autocomplete and I can't figure out how to actually get a result using "is" and Team members